hypha/apply/funds/templates/funds/includes/screening_status_block-button.html

```diff
@@ -0,0 +1,90 @@
+{% load i18n heroicons %}
+
+<div class="relative"
+     x-data="{ isOpen: false, openedWithKeyboard: false, leaveTimeout: null }"
+     @mouseleave.prevent="leaveTimeout = setTimeout(() => { isOpen = false }, 250)"
+     @mouseenter="leaveTimeout ? clearTimeout(leaveTimeout) : true"
+     @keydown.esc.prevent="isOpen = false, openedWithKeyboard = false"
+     @click.outside="isOpen = false, openedWithKeyboard = false"
+>
+    <button
+        class="p-2 w-full h-full inline-flex flex-col items-center rounded-lg transition-colors text-sm
+               {% if current_status %} {% if type == "yes" %}button--like-active bg-light-blue/10 {% else %} button--dislike-active bg-red-100/80{% endif %}  font-bold{% elif can_screen %}hover:bg-slate-100{% endif %}"
+        {% if can_screen and screening_options.count > 1 %}
+            @mouseover="isOpen = true"
+            @focus="openedWithKeyboard = true"
+            @keydown.space.prevent="openedWithKeyboard = true"
+            @keydown.enter.prevent="openedWithKeyboard = true"
+            @keydown.down.prevent="openedWithKeyboard = true"
+        {% endif %}
+        hx-post="{% url 'funds:submissions:partial-screening-card' submission.id %}"
+        hx-swap="outerHTML"
+        hx-trigger="click"
+        hx-target="#screening-status-{{ submission.id }}"
+        hx-vals='{"action": "{% if not current_status %}{{ default_status.id }}{% else %}clear{% endif %}"}'
+        {% if current_status %}
+            hx-confirm='Are you sure you want to remove the "{{ current_status }}" screening decision?'
+            data-tippy-content="Remove"
+        {% else %}
+            data-tippy-content="Mark as {{ default_status }}"
+        {% endif %}
+    >
+        {% if type == "yes" %}
+            {% heroicon_solid "hand-thumb-up" size=30 class="mb-2 transition-colors" aria_hidden=true %}
+        {% else %}
+            {% heroicon_solid "hand-thumb-down" size=30 class="mb-2 transition-colors" aria_hidden=true %}
+        {% endif %}
+        {{ current_status|default_if_none:default_status }}
+    </button>
+
+    <!-- panel -->
+    {% if can_screen and screening_options.count > 1 %}
+        <div class="bg-white min-w-44 text-sm absolute start-0 end-0 divide-y rounded border shadow-lg group-hover:absolute z-20"
+             x-cloak
+             role="menu"
+             x-show="isOpen || openedWithKeyboard"
+             x-transition.origin.top
+             x-trap="openedWithKeyboard"
+             @click.outside="isOpen = false, openedWithKeyboard = false"
+             @keydown.down.prevent="$focus.wrap().next()"
+             @keydown.up.prevent="$focus.wrap().previous()"
+        >
+            <header class="subheading border-b px-3 py-2 flex justify-between items-center gap-2">
```

Comment:
py-n classess add padding-right/left. Better to use classes that add padding-inline-start/end.